app/controllers/api/articles_controller.rb

```diff
@@ -1,19 +1,43 @@
+# frozen_string_literal: true
+
 class Api::ArticlesController < ApplicationController
   def index
     collection_articles = Article.all
     if collection_articles.empty?
-      render json: { error_message: 'No articles has been found' }, status: 404
+      render json: { message: 'No articles has been found' }, status: 404
     else
       render json: { articles: collection_articles }, status: 200
    end
   end
 
   def show
-    article = Article.find(params[:id]) rescue nil
+    article = begin
+                Article.find(params[:id])
+              rescue StandardError
+                nil
+              end
     if article.nil?
-      render json: { message: "Sorry your article wasn't found" }, status: 404 
+      render json: { message: "Sorry your article wasn't found" }, status: 404
     else
       render json: article, status: 200
     end
   end
+
+  def create
+    article = Article.new(article_params)
```

Comment:
You can (and should) avoid nested `if` statements.

How about calling `Article.create` and use `article.persisted?` in the `if` statement? Then you can skip the `.new_record?` check.